tests/driver_dfplayer/Makefile

```diff
@@ -0,0 +1,11 @@
+include ../Makefile.tests_common
+
+BUSY_PIN ?= GPIO_PIN(0,8)
+
+USEMODULE += dfplayer
+USEMODULE += shell
+USEMODULE += shell_commands
+
+include $(RIOTBASE)/Makefile.include
+
+CFLAGS += '-DDFPLAYER_PARAM_BUSY_PIN=$(BUSY_PIN)'
```

Comment:
This will cause problems if `DFPLAYER_PARAM_BUSY_PIN` is already defined in `board.h`.

Reply:
True. I'll drop the definition of `BUSY_PIN` and only extent `CFLAGS` if `BUSY_PIN` is defined.